support.md

```diff
@@ -11,7 +11,7 @@ If you need any support either
 * send an email with your question to [support@tradecloud1.com](mailto:support@tradecloud.nl)
 * using the Intercom widget on the bottom right hand corner of the Tradecloud portal \(log in first\).
 
-API support is best effort, usually you will receive a response within 2 hours with a maximum of 24 hours on working hours in the CET timezone.
```

Comment:
We do no support in the summer any more?

Reply:
We do, that's why it has an `(S)` now in it.

Reply:
Wauw, my mistake, I have to read from left to right

Reply:
And cancel my headache